Applications/README.md

```
@@ -0,0 +1,221 @@
+- **Maven** 3.6+
+- **Node.js** 16+ and **npm**
+- **Angular CLI** (`npm install -g @angular/cli`)
+- **tmux**
+- **Git**
+- **wget**, **unzip**, and **mysql** CLI (used by the master data loader)
+
+---
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)